sw/snRuntime/src/dma.c

```diff
@@ -52,8 +52,8 @@ snrt_dma_txid_t snrt_dma_start_1d(void *dst, const void *src, size_t size) {
 
 /// Initiate an asynchronous 2D DMA transfer with wide 64-bit pointers.
 snrt_dma_txid_t snrt_dma_start_2d_wideptr(uint64_t dst, uint64_t src,
-                                          size_t size, size_t dst_stride,
-                                          size_t src_stride, size_t repeat) {
+                                          size_t size, size_t src_stride,
```

Comment:
this change seems a bit counterintuitive; is there a reason why the strides should not have the same order as the addresses?

Reply:
There is an inconsistency between `snrt_dma_start_2d` and `snrt_dma_start_2d_wideptr`. The former calls the latter, but `dst_stride` and `src_stride` are swapped in the process. Maybe it makes more sense to change it in `snrt_dma_start_2d` and always take the `dst` arguments before `src` arguments to be consistent